pyvista/core/filters.py

```diff
@@ -3566,3 +3566,24 @@ def gaussian_smooth(dataset, radius_factor=1.5, std_dev=2.,
             alg.SetStandardDeviations(std_dev, std_dev, std_dev)
         alg.Update()
         return _get_output(alg)
+
+
+    def extract_subset(dataset, voi):
+        """Select piece (e.g., volume of interest).
+
+        To use this filter set the VOI ivar which are i-j-k min/max indices
+        that specify a rectangular region in the data. (Note that these are
+        0-offset.) You can also specify a sampling rate to subsample the
+        data.
+
+        Parameters
+        ----------
+        voi : tuple(int)
+            Length 6 tuple of integers specifying the volume of interest in
+            i-j-k min/max indices.
```

Comment:
I'd add an illustration of the ordering here, like in
https://github.com/pyvista/pyvista/blob/a572fd76cc5fac46a2f7e8c5ce977edd35ed1175/pyvista/core/filters.py#L130

Reply:
done